### ISSUE TITLE
[Improvement]: Default Value for Bool in Advanced ManyToMany Relation Column #10524

### ISSUE THREAD

**Issue by evanjamesjackson** [TorqIT/pimcore#9]
https://github.com/pimcore/pimcore/issues/10524

**Comment by lukemacausland** [comment]
Did some testing around this, and it looks like other default values for the other columns appear to not be set as well. A user actually commented on this further down as well. I think it would be beneficial to quickly hook these up properly too.

https://github.com/pimcore/pimcore/issues/10524#issuecomment-1172131639

Talked with Evan, we should maybe just open up a new issue for the others.

**Comment by evanjamesjackson** [comment]
Blocked by https://github.com/TorqIT/pimcore-testbed/issues/2
